### PR TITLE
minor: regenerate JSON files

### PR DESCRIPTION
**Pull request by kmahar**
I am not sure why, but when I regenerated files after making some changes in #1324 these ones changed. Seems like they had not been properly regenerated for whatever reason and the GitHub Action checking the files did not pick it up.

### CODE REVIEW

**Review by nbbeeken — APPROVED** [review]

**Comment by BorisDog** [comment]
Looks like this is related to [DRIVERS-1707](https://github.com/mongodb/specifications/pull/1333) cc @Neilshweky @DmitryLukyanov

File: source/server-discovery-and-monitoring/tests/unified/interruptInUse-pool-clear.json [hunk 1]
```diff
@@ -54,7 +47,14 @@
                     "commandFailedEvent",
                     "connectionCheckedOutEvent",
                     "connectionCheckedInEvent"
-                  ]
+                  ],
```
Comment (DmitryLukyanov) [inline source/server-discovery-and-monitoring/tests/unified/interruptInUse-pool-clear.json:50]: some minor formatting fix was applied yesterday, but I would not expect such change in order..

File: source/server-discovery-and-monitoring/tests/unified/interruptInUse-pool-clear.json [hunk 2]
```diff
@@ -92,44 +94,47 @@
...
-                "object": "collection",
-                "arguments": {
-                    "filter": { "$where": "sleep(2000) || true" }
-                },
-                "expectError": {
-                  "isError": true
+              "name": "find",
+              "object": "collection",
+              "arguments": {
+                "filter": {
```
Comment (DmitryLukyanov) [inline source/server-discovery-and-monitoring/tests/unified/interruptInUse-pool-clear.json:100]: I don't understand why this change was applied, but probably it's not a big deal

**Review by DmitryLukyanov — APPROVED** [review]
Looks like no real changes, so LGTM

File: source/server-discovery-and-monitoring/tests/unified/interruptInUse-pool-clear.json (hunk 2 above)
Comment (jmikola) [reply source/server-discovery-and-monitoring/tests/unified/interruptInUse-pool-clear.json:100]: `js-yaml` does not inline JSON objects. By default, keys within an object appear indented on their own line between the opening and closing braces.
